projects/plugins/jetpack/class.json-api.php

```diff
@@ -1021,26 +1021,38 @@ public function wp_count_comments( $post_id ) {
 		);
 
 		/**
-		 * Exclude certain comment types from comment counts in the REST API.
-		 *
-		 * @since 6.9.0
-		 * @module json-api
-		 *
-		 * @param array Array of comment types to exclude (default: 'order_note', 'webhook_delivery', 'review', 'action_log')
-		 */
-		$exclude = apply_filters(
-			'jetpack_api_exclude_comment_types_count',
-			array( 'order_note', 'webhook_delivery', 'review', 'action_log' )
+		* Exclude certain comment types from comment counts in the REST API.
+		*
+		* @deprecated $$next-version$$
+		* @module json-api
+		*
+		* @param array Array of comment types to exclude (default: 'order_note', 'webhook_delivery', 'review', 'action_log')
+		*/
+
+		$exclude = apply_filters_deprecated( 'jetpack_api_exclude_comment_types_count', array( 'order_note', 'webhook_delivery', 'review', 'action_log' ), '10.9.1', 'jetpack_api_include_comment_types_count' );
```

Comment:
```suggestion
		$exclude = apply_filters_deprecated( 'jetpack_api_exclude_comment_types_count', array( 'order_note', 'webhook_delivery', 'review', 'action_log' ), 'jetpack-11.0', 'jetpack_api_include_comment_types_count' ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Doesn't seem like `$$next-version$$` will work in the `apply_filters_deprecated`, I'll have to take a look at that later.